app/models/ebean/Organization.java

```diff
@@ -0,0 +1,143 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package models.ebean;
+
+import com.avaje.ebean.Model;
+import com.avaje.ebean.annotation.CreatedTimestamp;
+import com.avaje.ebean.annotation.UpdatedTimestamp;
+
+import java.sql.Timestamp;
+import java.util.UUID;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.Version;
+
+/**
+ * Data model for organizations.
+ *
+ * @author Brandon Arp (brandon dot arp at smartsheet dot com)
+ */
+// CHECKSTYLE.OFF: MemberNameCheck
+@Entity
+@Table(name = "organizations", schema = "portal")
+public class Organization extends Model {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "uuid")
+    private UUID uuid;
+
+    @Version
+    @Column(name = "version")
+    private Long version;
+
+    @CreatedTimestamp
+    @Column(name = "created_at")
+    private Timestamp createdAt;
+
+    @UpdatedTimestamp
+    @Column(name = "updated_at")
+    private Timestamp updatedAt;
+
+    @Column(name = "name")
+    private String name;
+
+    /**
+     * Finds an {@link Organization} when given an {@link models.internal.Organization}.
+     *
+     * @param organization The organization to lookup.
+     * @return The organization from the database.
+     */
+    @Nullable
+    public static Organization findByOrganization(@Nonnull final models.internal.Organization organization) {
+        final Organization org = FINDER
+                .where()
+                .eq("uuid", organization.getId())
+                .findUnique();
+        return org;
+    }
+
+    /**
+     * Returns an {@link Organization} by reference. This defers all database operations until a field besides id is
+     * requested.
+     *
+     * @param id The id (primary key) of the organization.
+     * @return The {@link Organization} with the given primary key.
+     */
+    public static Organization refById(final long id) {
+        return FINDER.ref(id);
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(final Long value) {
+        id = value;
+    }
+
+    public Long getVersion() {
+        return version;
+    }
+
+    public void setVersion(final Long value) {
```

Comment:
If we're using long version on the entities we should definitely use it on the etag table too.